Demo/DemoTests/MoyaProviderSpec.swift

```diff
@@ -277,7 +277,7 @@ class MoyaProviderSpec: QuickSpec {
                         let parallelCount = 10
                         let observables = Array(0..<parallelCount).map { _ in provider.request(target) }
                         var completions = Array(0..<parallelCount).map { _ in false }
-                        let queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)
+                        let queue = dispatch_queue_create("testing", DISPATCH_QUEUE_CONCURRENT)
```

Comment:
Nah, no problem! I had to look it up, too. `dispatch_release` isn't even exposed in Swift.